extension/src/popup/components/manageNetwork/NetworkIcon/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import "./styles.scss";
+
+/*
+  Custom network icon colors are determined by where they fall in the list. We cycle through 4 colors and repeat if we get to the end. 
+*/
```

Comment:
I don't love the idea of having a different color for every Custom network. I also don't really like the idea of the color being determined by where it falls in the list.

But I really really don't like the idea of saving a Custom network's color in the "database". So I created this slightly over-engineered component to figure out a Network icon's color based on where it falls in the list of networks